_uk/index.md

```diff
@@ -0,0 +1,128 @@
+        description: "Встанови Scala 3 на свій комп'ютер і почни писати код Scala!"
+        icon: "fa fa-rocket"
+        link: /scala3/getting-started.html
+      - title: "Книга по Scala 3"
+        description: "Вивчи Scala, прочитавши серію коротких уроків."
+        icon: "fa fa-book-open"
+        link: /scala3/book/introduction.html
+  - title: "Детальніше"
+    links:
+      - title: "Посібник з міграції"
+        description: "Посібник, що допоможе перейти від Scala 2 до Scala 3."
+        icon: "fa fa-suitcase"
+        link: /scala3/guides/migration/compatibility-intro.html
+      - title: "Посібники"
+        description: "Детальні посібники про окремі аспекти мови."
+        icon: "fa fa-map"
+        link: /scala3/guides.html
+      - title: "API бібліотек Scala"
+        description: "Документація API для кожної версії стандартної бібліотеки Scala 3."
+        icon: "fa fa-file-alt"
+        link: https://scala-lang.org/api/3.x/
+      - title: "Довідник про мову"
+        description: "Довідкова інформація про мову Scala 3."
+        icon: "fa fa-book"
+        link: https://docs.scala-lang.org/scala3/reference
+      - title: "Зробити внесок у Scala 3"
+        description: "Посібник з компілятора Scala 3 та вирішення проблем."
+        icon: "fa fa-cogs"
+        link: /scala3/guides/contribution/contribution-intro.html
+      - title: "Новий Scaladoc для Scala 3"
+        description: "Основні характеристики нових функцій у Scaladoc."
+        icon: "fa fa-star"
+        link: /scala3/scaladoc.html
+      - title: "Доповіді"
+        description: "Онлайн доповіді про Scala 3."
+        icon: "fa fa-play-circle"
+        link: /scala3/talks.html
+      - title: "Онлайн курси"
+        description: "MOOC для вивчення Scala для початківців і досвідчених програмістів."
+        icon: "fa fa-cloud"
+        link: /online-courses.html
+
+scala2-sections:
+  - title: "Перші кроки..."
+    links:
+      - title: "Початок роботи"
+        description: "Встанови Scala на свій комп'ютер і почни писати код Scala!"
+        icon: "fa fa-rocket"
+        link: /getting-started.html
+      - title: "Екскурсія по Скала"
+        description: "Короткі введення в основні особливості мови."
+        icon: "fa fa-flag"
+        link: /tour/tour-of-scala.html
+      - title: "Книга по Scala"
+        description: "Вивчи Scala, прочитавши серію коротких уроків."
+        icon: "fa fa-book-open"
+        link: /overviews/scala-book/introduction.html
+      - title: Онлайн курси
+        description: "MOOC для вивчення Scala для початківців і досвідчених програмістів."
+        icon: "fa fa-cloud"
+        link: /online-courses.html
+      - title: Книги
+        description: "Друковані та цифрові книги по Scala."
+        icon: "fa fa-book"
+        link: /books.html
+      - title: Посібники
+        description: "Путівник з серії кроків для створення програм на Scala."
+        icon: "fa fa-tasks"
+        link: /tutorials.html
```

Comment:
Why previous titles were written in quotes and these ones without?

Reply:
Missed them. Fixed